compiler/test/test_wasm_utils.re

```diff
@@ -80,18 +70,18 @@ let test_leb128_i32 = ctxt => {
 };
 
 let test_get_wasm_sections = ctxt => {
-  ignore(compile_wasm_file("testmod"));
```

Comment:
I just went ahead and checked in a compiled version of this file (compiled via wat2wasm) and it sits next to the source.